### PR TITLE
🌙 Support for Moonbase Alpha chain (testnet on Moonbeam)

### PR DESCRIPTION
**Pull request by wagmiwiz**
https://docs.moonbeam.network/builders/get-started/moonbase/

### CODE REVIEW

File: packages/core/src/helpers/chain.ts [hunk 1]
```diff
@@ -79,6 +81,8 @@ export function getExplorerTransactionLink(transactionHash: string, chainId: Cha
       return `https://ftmscan.com/tx/${transactionHash}`
     case ChainId.Avalanche:
       return `https://snowtrace.io/tx/${transactionHash}`
+    case ChainId.MoonbaseAlpha:
+      return `https://moonbase.moonscan.io/tx/${transactionHash}`
```
Comment (tt-marek) [inline packages/core/src/helpers/chain.ts:85]: Is it multicall contract?
Can't see code:
https://moonbase.moonscan.io/address/0x4E2cfca20580747AdBA58cd677A998f8B261Fc21#code

Comment (tt-marek) [reply packages/core/src/helpers/chain.ts:85]: @wagmiwiz Would you verify it?

Comment (albertov19) [reply packages/core/src/helpers/chain.ts:85]: @tt-marek I've verified the contract :) - https://moonbase.moonscan.io/address/0x4E2cfca20580747AdBA58cd677A998f8B261Fc21#code

Comment (tt-marek) [reply packages/core/src/helpers/chain.ts:85]: ❤️

**Comment by tt-marek** [comment]
❤️ Looks great, would you rebase/merge master?

**Comment by wagmiwiz** [comment]
Done @tt-marek

File: packages/core/test/helpers/chain.test.ts [hunk 2]
```diff
@@ -65,8 +67,8 @@ describe('Chain helpers', () => {
     expect(isLocalChain(ChainId.Palm)).to.be.false
     expect(isLocalChain(ChainId.Fantom)).to.be.false
     expect(isLocalChain(ChainId.Avalanche)).to.be.false
-    expect(isTestChain(ChainId.Fantom)).to.be.false
-    expect(isTestChain(ChainId.Songbird)).to.be.false
+    expect(isLocalChain(ChainId.Songbird)).to.be.false
+    expect(isLocalChain(ChainId.MoonbaseAlpha)).to.be.false
```
Comment (marekkirejczyk) [inline packages/core/test/helpers/chain.test.ts:71]: I think during the merge Fantom was removed

**Review by marekkirejczyk — CHANGES_REQUESTED** [review]

**Comment by wagmiwiz** [comment]
Yeah, it looks like previous commit has a copy paste error so this is
correcting. Fantom is already being checked for isLocal

**Review by marekkirejczyk — APPROVED** [review]

**Pull request merged**